app/models/user.rb

```diff
@@ -66,7 +66,7 @@ def confirmed_email_addresses
     email_addresses.where.not(confirmed_at: nil).order('last_sign_in_at DESC NULLS LAST')
   end
 
-  def need_two_factor_authentication?(_request)
```

Comment:
I think this is overriding a method from Devise... totally OK if they share implementations but I think we need to keep it around

Reply:
This was originally a method we were originally overriding as part of using `two_factor_authentication` gem, which we removed in #4456.